researcher_ids/RI_Claims_V1.md

```diff
@@ -1194,16 +1355,24 @@ reader-friendly.
             "value": "https://ega-archive.org/datasets/00000432",
             "source": "https://grid.ac/institutes/grid.225360.0",
```

Comment:
```suggestion
            "source": "https://ega-archive.org/dacs/EGAC00001000205",
```
A DAC identifier for the dataset above.

Reply:
The EGA service of EBI does not run any DACs. The DACs are nominated by the data owners. So the Passport Visa Assertion Source is the DAC although the Passport Visa Assertion Repository and Signatory are probably EBI.

Reply:
Done